lake_viewer.py

```diff
@@ -291,41 +260,11 @@ def local_css(file_name):
 def remote_css(url):
     st.markdown(f'<link href="{url}" rel="stylesheet">', unsafe_allow_html=True)
 
-def getJS():
-    myjs = '''<script>
-        /* Loop through all dropdown buttons to toggle between hiding and showing its dropdown content - This allows the user to have multiple dropdowns without any conflict */
-
-        var dropdown =  window.parent.document.getElementsByClassName("dropdown-btn");
-        var i;
-        for (i = 0; i < dropdown.length; i++) {
-        dropdown[i].addEventListener("click", function() {
-            this.classList.toggle("active");
-            var dropdownContent = this.nextElementSibling;
-            if (dropdownContent.style.display === "block") {      
-            dropdownContent.style.display = "none";
-            } else {      
-            dropdownContent.style.display = "block";
-            }
-        });
-        }
-        </script>'''
-    return myjs
-
 local_css("lv.css")
 remote_css('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css')
 
 def main():
-    ns = st.query_params.get_all('namespace')
-    tb = st.query_params.get_all('table')
-    partition = st.query_params.get_all('partition')
-    sample_limit = st.query_params.get_all('sample_limit')
-    limit = 100
```

Comment:
Question - what mechanism we can provide such that one can share direct link to a table?

Reply:
Let me find a way